_internals/tds-protocol.md

```diff
@@ -85,11 +85,31 @@ Once we are over the <code>PRELOGIN</code> phase, we are ready for
 <code>LOGIN7</code> which defines the authentication rules for use 
 between client and server.
 
+### SSL/TLS support for TDS connections
 
-### Defining authentication rules.
+Babelfish uses native PostgreSQL support for using SSL connections to encrypt client/server communications. See PostgreSQL [documentation](https://www.postgresql.org/docs/13/ssl-tcp.html#SSL-SETUP) for details about configuring server-side SSL functionality.
```

Comment:
The start of the second sentence is missing the word 'the' - it should be 'See the PostgreSQL documentation...'